### PR TITLE
metrics: Add checkmetrics to gha-run.sh for metrics CI

### LINKED ISSUE #7198: add checkmetrics to gha-run.sh for metrics

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds checkmetrics installation for gha-run.sh in order to compare results limits as part of the metrics CI.

Fixes #7198

### CODE REVIEW

**Review by dborquez — APPROVED** [review]

File: tests/metrics/cmd/checkmetrics/ci_worker/checkmetrics-json-qemu-kata-metric8.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file contains baseline expectations
+# for checked results by checkmetrics tool.
+#
+# values set specifically for packet.com c1.small worker.
```
Comment (chavafg) [inline tests/metrics/cmd/checkmetrics/ci_worker/checkmetrics-json-qemu-kata-metric8.toml:8]: we changed the worker the server type to `m3.small.x86`. Let's also change packet.com to Equinix (new name)

File: tests/metrics/gha-run.sh [hunk 2]
```diff
@@ -65,6 +67,15 @@ function install_kata() {
 
 	check_containerd_config_for_kata
 	restart_containerd_service
+	install_checkmetrics
+}
+
+function install_checkmetrics() {
+	# Ensure we have the latest checkemtrics
```
Comment (chavafg) [inline tests/metrics/gha-run.sh:74]: typo: `checkmetrics`

File: tests/metrics/gha-run.sh [hunk 3]
```diff
@@ -73,23 +84,34 @@ function check_containerd_config_for_kata() {
...
 	fi
 }
 
+function check_metrics() {
+	KATA_HYPERVISOR="${1}"
+	local cm_base_file="${checkmetrics_config_dir}/checkmetrics-json-${KATA_HYPERVISOR}-kata-metric8.toml"
+	checkmetrics --debug --percentage --basefile "${cm_base_file}" --metricsdir "${results_dir}"
+	cm_result=$?
+	if [ "${cm_result}" != 0 ]; then
+		info "run-metrics-ci: checkmetrics FAILED (${cm_result})"
```
Comment (chavafg) [inline tests/metrics/gha-run.sh:103]: I think we can change `info` to `die` and remove line below

Comment (GabyCT) [reply tests/metrics/gha-run.sh]: @chavafg changes applied

**Review by chavafg — APPROVED** [review]

**Pull request merged**